src/indigo/components/StatusCircle.js

```diff
@@ -4,16 +4,15 @@ import classnames from 'classnames';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faCircle } from '@fortawesome/free-solid-svg-icons/faCircle';
 import { faCheckCircle } from '@fortawesome/free-solid-svg-icons/faCheckCircle';
-import { faTimesCircle } from '@fortawesome/free-solid-svg-icons/faTimesCircle';
 import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons/faExclamationCircle';
 
 const statusMap = {
   success: { status: 'success', icon: faCheckCircle },
   error: { status: 'danger', icon: faExclamationCircle },
-  warn: { status: 'danger', icon: faTimesCircle },
-  warning: { status: 'danger', icon: faTimesCircle },
   processing: { status: 'warning', icon: faCircle },
   waiting: { status: 'default', icon: faCircle },
+  warn: { status: 'warning', icon: faExclamationCircle },
```

Comment:
Jen pro zajímavost, proč se to přesunulo? 😄

Reply:
Aby ikonky byly srovnaný :D nejdřív faCircle pak faExclamationCircle 
Asi nemoc no :D

Reply:
I když vidím beztak error je nahoře, takže se to nepovedlo komplet.